src/phantasm-hardware-interface/common/sse_vec.hh

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <clean-core/macros.hh>
+
+#include <emmintrin.h>
+
+namespace phi
```

Comment:
why is this public API of phi? is it used in some other lib as well?

Reply:
moved to phi::util